inst/@sym/mrdivide.m

```diff
@@ -27,27 +27,7 @@
 
 function z = mrdivide(x, y)
 
-  % Dear hacker from the distant future... maybe you can delete this?
-  if (isa(x, 'symfun') || isa(y, 'symfun'))
-    warning('OctSymPy:sym:arithmetic:workaround42735', ...
-            'worked around octave bug #42735')
-    z = mrdivide(x, y);
-    return
-  end
-
-
-  if isscalar(x) && isscalar(y)
-    z = rdivide(x, y);
-
-  elseif isscalar(x) && ~isscalar(y)
-    error('FIXME: scalar/array not implemented yet');
-
-  elseif ~isscalar(x) && isscalar(y)
-    z = rdivide(x, y);
-
-  else  % two array's case
-    error('FIXME: array/array not implemented yet');
-  end
+  z = rdivide(x, y);
```

Comment:
https://en.wikipedia.org/wiki/Division_%28mathematics%29#Of_matrices

Unlike Matlab, Octave doesn't seem to have a nice page documenting this... but basically `x = A \ b` means solve `A*x = b`.  And `X = B \ A` mean solve `X*A = B` (i.e., `C = A*inv(B)`).

But yes probably the error message could be improved: I'll file an issue to do that.